src/main/java/org/gridsuite/modification/server/entities/ModificationEntity.java

```diff
@@ -86,6 +86,10 @@ public void update(ModificationInfos modificationInfos) {
         }
     }
 
+    public ModificationEntity copy() {
```

Comment:
Maybe use the Cloneable interface instead ?

Reply:
When you implement clone method in Java you need to call first super.clone
Hereis just a copy of an entity and a clone of the Java object

Reply:
Ok, if it's on purpose, maybe call it shallowCopy instead, then people who will use it will be sure it's not a deep copy.